HomeWork/Home3.2/src/main/java/com/example/Triangle.java

```diff
@@ -0,0 +1,31 @@
+package com.example;
+
+public class Triangle {
+    private double sideA;
+    private double sideB;
+    private double sideC;
+
+    public Triangle(double sideA, double sideB, double sideC) {
+        this.sideA = sideA;
+        this.sideB = sideB;
+        this.sideC = sideC;
+    }
+
+    public double getHeight(char side) {
```

Comment:
it is better to use enum as an argument. so validation of the inputs will be done at higher level